### PR TITLE
HA: Set Optional ordering for Glance startup

### PR DESCRIPTION
**Pull request by dirkmueller**
Let Glance depend on Keystone, Database and RabbitMQ to be
started first.

### CODE REVIEW

File: chef/cookbooks/glance/recipes/ha.rb [hunk 1]
```diff
@@ -74,4 +74,11 @@
   only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
 end
 
+crowbar_pacemaker_order_only_existing "o-#{group_name}" do
```
Comment (nkrinner) [inline chef/cookbooks/glance/recipes/ha.rb]: Shouldn't that just be
 ``pacemaker_order "o-o-#{group_name}" do`` ?

Comment (vuntz) [inline chef/cookbooks/glance/recipes/ha.rb]: Hrm, why?

File: chef/cookbooks/glance/recipes/ha.rb [hunk 2]
```diff
@@ -74,4 +74,11 @@
...
+  ordering "postgresql rabbitmq cl-keystone cl-#{group_name}"
```
Comment (vuntz) [inline chef/cookbooks/glance/recipes/ha.rb]: Is keystone really needed?

File: chef/cookbooks/glance/recipes/ha.rb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/glance/recipes/ha.rb]: Should be `o-cl-#{group_name}` though (maybe that's what you wanted to type ;-))

File: chef/cookbooks/glance/recipes/ha.rb (hunk 2 above)
Comment (vuntz) [inline chef/cookbooks/glance/recipes/ha.rb]: (Probably doesn't matter much if we change to `Optional`)

**Comment by vuntz** [comment]
Submitted https://github.com/dirkmueller/barclamp-glance/pull/1 for my fixes

**Comment by vuntz** [comment]
+1

**Pull request merged**